src/scripts/clipperUI/components/modeButtonSelector.tsx

```diff
@@ -1,28 +1,39 @@
-import {ClientType} from "../../clientType";
 import {Constants} from "../../constants";
-import {Experiments} from "../../experiments";
-
 import {AugmentationHelper} from "../../contentCapture/augmentationHelper";
-
 import {ExtensionUtils} from "../../extensions/extensionUtils";
 import {InvokeMode} from "../../extensions/invokeOptions";
-
 import {Localization} from "../../localization/localization";
-
 import {ClipMode} from "../clipMode";
 import {ClipperStateProp} from "../clipperState";
 import {ComponentBase} from "../componentBase";
-
 import {ModeButton} from "./modeButton";
 
+export interface NoAriaPropsForModeButton {
+	imgSrc: string;
+	label: string;
+	myMode: ClipMode;
+	selected?: boolean;
+	tabIndex?: number;
+	onModeSelected: (modeButton: ClipMode) => void;
+	tooltipText?: string;
+}
+
+export interface AriaPropsForModeButton {
```

Comment:
This should belong in another file, since it has nothing to do with modebuttons anymore (abstraction)

Reply:
Also the name should not reference ModeButton at all, since it's generalized enough to be reused